Jenkinsfile

```diff
@@ -31,17 +36,34 @@ pipeline {
           steps {
             println "Stage running on ${env.NODE_NAME}"
 
+            get_xcommon_cmake()
+
             dir("${REPO}") {
               checkout scm
 
+              withTools("15.2.1") {
```

Comment:
To reduce change for forgetting this one I think we should add PREV_TOOLS_VERSION to environment